test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -442,8 +442,16 @@ class test_ucp_wireup_1sided : public test_ucp_wireup {
     }
 
     ucp_object_version_t address_version() const {
-        return (get_variant_value() & WORKER_ADDR_V2) ?
-               UCP_OBJECT_VERSION_V2 : UCP_OBJECT_VERSION_V1;
+        if (get_variant_value() & WORKER_ADDR_V2) {
+            return UCP_OBJECT_VERSION_V2;
+        }
+
+        const char *str = getenv("UCX_ADDRESS_VERSION");
```

Comment:
this is to allow running gtest on system with more than 31 MD (md_index > 31)

Reply:
can you pls add a comment to explain? maybe we should instantiate only the v2 variant, or ignore extra MDs with addrv1?

Reply:
Ignoring more than 31 md in address packing might be difficult as there might be cases where we don't want to restrict the number of md_index, so this might be another option to add in UCP? Maybe eventually limit them from UCT, in EFA? But since interfaces have names, some could be down only on some nodes, so I guess we cannot just a count like `UCX_EFA_IFACES=3`?
Added comment in the meantime.

Reply:
we could count **active** interface in UCT/EFA and limit the amount of resources returned from UCT query, according to a new env var UCX_EFA_MAX_IFACES

Reply:
I did not find proper way to add such config because:
1. efa md is under common ib component, and currently does not implement its own transport
2. `uct_ib_query_md_resources()`/`ibv_get_device_list()` are called with component as a parameter, and there is no md yet/md config passed

Do we want to add configuration parsing tied to component instances?

Reply:
yes, to the component